sources/ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -100,6 +100,7 @@ public static int Run(InvocationContext context)
 
             var clangCommandLineArgs = new string[]
             {
+                "-std=c++11",                           // The input files should be compiled for C++ 11
```

Comment:
Clang 5 (which is currently being used for tests) doesn't target C++11 by default yet.